### ISSUE TITLE
Integrity constraint violation: 1062 Duplicate entry

### ISSUE THREAD

**Issue by Metis77** [codefog/contao-instagram#33]
After updating from an older v1 version of this plugin we get this error:

```
Integrity constraint violation: 1062 Duplicate entry '53' for key 'PRIMARY'" at /var/www/vhosts/***/httpdocs/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 74 null []
```

We could get a Instagram Access Token and the images are sucessfully downloaded.
But the frontend generates this error.

**Comment by qzminski** [comment]
Would you be able to open the website in dev mode and shed some more light on the query that fails?

**Comment by Metis77** [comment]
this links to the page with dev mode enabled, for now:
https://dionysius.ch.83-169-33-59.wolke7.hauptwolke.de/
Does this help?

**Comment by Metis77** [comment]
Does this help?
Would you need anything else?

![dionysius ch 83-169-33-59 wolke7 hauptwolke de_ (1)](https://user-images.githubusercontent.com/1408361/103637978-915f9000-4f4c-11eb-98a2-b5b8fd89589a.png)

**Comment by qzminski** [comment]
Can you please try `dev-hotfix/2.2.2` branch and see if 5f7b256 solves your problem?

**Comment by Metis77** [comment]
wow thanks for that.
let me know, I we can support / fund this project a little 

sorrily we now get another error:

<img width="1041" alt="Bildschirmfoto 2021-01-07 um 12 06 54" src="https://user-images.githubusercontent.com/1408361/103885959-124f9080-50e1-11eb-8b93-15f0ade16a48.png">

<img width="1080" alt="Bildschirmfoto 2021-01-07 um 12 07 19" src="https://user-images.githubusercontent.com/1408361/103885984-1bd8f880-50e1-11eb-8653-5ea89366a135.png">

**Comment by qzminski** [comment]
I am not sure what's wrong with the model registry, but 1dfd90e81e5a5f800c8b1855c64e65750afed225 should fix it now 😉

**Comment by qzminski** [comment]
I have just released 2.2.2 stable :+1:
